tests/widget/modal/widget-modal.login-social-redirect.test.js

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from '@playwright/test';
+
+test.only('Modal widget with social callback redirect', async ({ page }) => {
+  await page.goto('widget/modal?journey=LoginWithSocial');
+
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  await loginButton.click();
+
+  await page.fill('text="User Name"', 'demouser');
+
+  await page.locator('button', { hasText: 'Continue with Google' }).click();
+
+  await page.waitForNavigation();
+  expect(page.url()).toContain('accounts.google.com', 'redirect_uri');
```

Comment:
When it comes to e2e testing for this I'm aware that some of it inherently needs to be manual since we are dealing with external services, accounts etc. However one bit that I thought could be auto tested was the redirect callback - to keep the test simple I've assumed that our test tenant will provide a Google social IDP and that we just use that to verify that the user is redirected to the Google accounts page as part of an OAuth flow. I originally looked at extracting the redirectURI from the AM response and verifying that we get sent there, however it felt like overkill in this test file and might bloat the tests, so instead went for this simpler test. Let me know what you think @cerebrl.

Reply:
Also I concluded that I don't think we should test the "return" part of the flow since the logic to identify the query params lives in the developer's app and the JS SDK itself, rather than any widget functionality. I opted to not test this.

Reply:
Anyway let me know if there are any obvious gaps in e2e that you find, and let's discuss. Thanks.

Reply:
Yeah, I think this is good. Let's make a note that the full flow needs to be manually tested. I think I have a section in the docs regarding use-cases that need to be manually tested. Can you add this to it?

Reply:
Yes, I will do. Thanks for reminding me.